dog-listing.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="zxx">
+<head>
+    <meta charset="utf-8">
+    <title>Adopt a Dog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+</head>
+<body>
+   <header>  
+     <img class="svg-dog-header" src="images/adoptadog-logo.svg" alt="Adopt-a-dog logo">
+     <p>Seattle's premiere dog adoption agency</p>
+     <div>
+       <img class="svg-cart" src="images/cart-icon.svg" alt="checkout cart icon">
+       <input class="cart-btn" type="submit" value="Checkout">
+     </div>
+   </header>
+   <main>
+      <nav class="nav-head">
+         <a class="nav-item-head" href="index.html">Home<span class="space">/</span></a>
+         <a class="nav-item-head" href="dog-listing.html">Dogs<span class="space">/</span></a>
+         <a class="nav-item-head" href="contact.html">Contact</a>
+     </nav>
+     <section class="dog-listing-container">
+        <h2 class="dog-listing-title">Here are all the Dogs!</h2>
```

Comment:
For your dog grid, I would recommend using CSS grid. Specifically using minMax, and auto-fill. Check out the guide here and see if you can replicate their grid on your site: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Realizing_common_layouts_using_CSS_Grid_Layout

Reply:
Thanks Tyler, 
I wanted to use Grid but James is suggesting Flex Box. 

I may do one with Flex box and the other with Grid.